.github/workflows/ci.yml

```diff
@@ -6,17 +6,23 @@ on:
 jobs:
   build:
     runs-on: ubuntu-24.04
-    container:
-      image: python:2.7.18
-    environment: ${{ (github.repository == 'jacoco/www.eclemma.org' && github.ref == 'refs/heads/master') && 'sites' || null }}
+    environment: ${{ (github.repository == 'jacoco/www.eclemma.org' && github.ref == 'refs/heads/snapshot') && 'sites' || null }}
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           fetch-depth: 0
       - name: 'Build'
-        run: ./render.sh
+        run: |
```

Comment:
Why do we need a docker container here? Is it due to python 2? In this case I can try to upgrade the rendering script to python 3.

Reply:
@marchof Cool! Thanks!
I propose to anyway review+merge this one first since it is ready and small one and will unblock publishing of actual snapshots to web site - without it we are deploying old snapshot, while #98 is bigger and will take me some time to review.

Reply:
No prob, I can rebase #98 afterwards.